### PR TITLE
IfcActuator Rail Import BIMQ 2021-07-21T06:26:28.4609038Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcActuator

### CODE REVIEW

**Review by jmirtsch — COMMENTED** [review]
Suggested changes.

File: IFC4x3/Properties/a/ActionBarMovement_3g9eB0$QrCGgwW7XmrxaeO/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ActionBarMovement_3g9eB0QrCGgwW7XmrxaeO" Name="ActionBarMovement" UniqueId="ea2682c0-fdad-4c42-aea0-1e1c35ee4a18" PrimaryDataType="IfcPositiveLengthMeasure" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/a/ActionBarMovement_3g9eB0$QrCGgwW7XmrxaeO/DocProperty.xml]: Done. Commmit : 23a962bd05b9764cc04293f6d2b700825df4893d

File: IFC4x3/Properties/c/Current_3zQJbuqOD69fAcKlNwK_SY/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The actual current and operable range.
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/c/Current_3zQJbuqOD69fAcKlNwK_SY/Documentation.md]: I have reused existing Current. 23a962bd05b9764cc04293f6d2b700825df4893d

File: IFC4x3/Properties/i/InstallationLimit_0_Xq_kDdTAXR3cFcN9ZofG/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Installation limit of the device.
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/i/InstallationLimit_0_Xq_kDdTAXR3cFcN9ZofG/Documentation.md]: Already inside Maintenance Pset. I have erased this one.

[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/i/InstallationLimit_0_Xq_kDdTAXR3cFcN9ZofG/Documentation.md]: 23a962bd05b9764cc04293f6d2b700825df4893d

File: IFC4x3/Properties/m/MarkingRodMovement_0yycL4CG92IuVgPQIqAk$Z/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MarkingRodMovement_0yycL4CG92IuVgPQIqAkZ" Name="MarkingRodMovement" UniqueId="3cf26544-3102-424b-87ea-65a4b42aefe3" PrimaryDataType="IfcPositiveLengthMeasure" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/m/MarkingRodMovement_0yycL4CG92IuVgPQIqAk$Z/DocProperty.xml]: Agree. 23a962bd05b9764cc04293f6d2b700825df4893d

File: IFC4x3/Properties/n/NumberOfTractionPoints_1Nk$_0PhT2kBz$hK6Rfjvq/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+The number of leverages or actuatrs needed to move turnouts.
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/n/NumberOfTractionPoints_1Nk$_0PhT2kBz$hK6Rfjvq/Documentation.md]: Done 23a962bd05b9764cc04293f6d2b700825df4893d

File: IFC4x3/Properties/c/ControlPulseCurrent_1NbOptutP7sPdNxheUF3ur/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+The current of the control pulse of the electric actuator.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/ControlPulseCurrent_1NbOptutP7sPdNxheUF3ur/Documentation.md]:
```suggestion
The current of the electric actuator control pulse.
```

File: IFC4x3/Properties/h/HasLockInside_1vFkVHKPjAvvxUxFl0WWv7/Documentation.md [hunk 7]
```diff
@@ -0,0 +1 @@
+Indicates whether the locking is inside of the point machine (TRUE) or outside of it (FALSE).
```
Comment (SergejMuhic) [inline IFC4x3/Properties/h/HasLockInside_1vFkVHKPjAvvxUxFl0WWv7/Documentation.md]:
```suggestion
Indicates whether the locking is inside  (TRUE) or outside (FALSE) of the point machine.
```

File: IFC4x3/Properties/l/LockingForce_0pLtPGaujDkwK16bnNbYZE/Documentation.md [hunk 8]
```diff
@@ -0,0 +1 @@
+Locking force of motor of the point machine.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LockingForce_0pLtPGaujDkwK16bnNbYZE/Documentation.md]:
```suggestion
Locking force of the point machine motor.
```

File: IFC4x3/Properties/m/MaximumOperatingTime_31nhczYx96JuYbkJ32Ejs4/Documentation.md [hunk 9]
```diff
@@ -0,0 +1 @@
+The maximum time the turnout has to complete the movement before the interlocking turns to out of control status.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MaximumOperatingTime_31nhczYx96JuYbkJ32Ejs4/Documentation.md]:
```suggestion
The maximum duration of the turnout movement before the interlocking turns to out of control status.
```

File: IFC4x3/Properties/m/MinimumOperatingSpeed_0OpoQ0$Az5huboLE7511z3/Documentation.md [hunk 10]
```diff
@@ -0,0 +1 @@
+Minimum operating speed of point machine.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MinimumOperatingSpeed_0OpoQ0$Az5huboLE7511z3/Documentation.md]:
```suggestion
Minimum operating speed of the point machine.
```

File: IFC4x3/Properties/n/NumberOfTractionPoints_1Nk$_0PhT2kBz$hK6Rfjvq/DocProperty.xml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfTractionPoints_1Nk_0PhT2kBzhK6Rfjvq" Name="NumberOfTractionPoints" UniqueId="57bbff80-66b7-42b8-bf7f-ad419ba6de74" PrimaryDataType="IfcCountMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfTractionPoints_1Nk$_0PhT2kBz$hK6Rfjvq/DocProperty.xml]: Seeing that this property is on actuator, is this not already possible to derive from the model?

File: IFC4x3/Properties/t/TractionForce_1h6R3D6qz8J9tT_sZdxp1y/Documentation.md [hunk 12]
```diff
@@ -0,0 +1 @@
+Traction force of point machine in turnout conversion.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TractionForce_1h6R3D6qz8J9tT_sZdxp1y/Documentation.md]:
```suggestion
Traction force of the point machine in turnout conversion.
```

File: IFC4x3/Properties/p/PointMachineType_0gfdQdlSPCaRPYyjTFQR6P/Documentation.md [hunk 13]
```diff
@@ -0,0 +1 @@
+Indicates the type of the railway point，different types of point allow trains to pass at different speeds。
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PointMachineType_0gfdQdlSPCaRPYyjTFQR6P/Documentation.md]:
```suggestion
Indicates the type of the railway point, different types of point allow trains to pass at different speeds.
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Properties/m/MarkingRodMovementLength_0yycL4CG92IuVgPQIqAk$Z/Documentation.md [hunk 14]
```diff
@@ -0,0 +1 @@
+The movement of the bar which indicates the turnout position.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/m/MarkingRodMovementLength_0yycL4CG92IuVgPQIqAk$Z/Documentation.md]:
```suggestion
The length of the movement bar which indicates the turnout position.
```

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]

**Review by AlexBrad1eyCT — APPROVED** [review]
All Sorted

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**